test/integration/enterprise_test.go

```diff
@@ -3,25 +3,32 @@
 package integration
```

Comment:
I've just noticed that the enterprise testing is co-opting the existing postgres tests. While I can see the overlap now, postgres and enterprise options are not mutually exclusive and so I feel we should maintain separate lineage for these and create a new test file for enterprise.

Reply:
This this API call https://github.com/Kong/kubernetes-testing-framework/pull/104/files#diff-937a1ace7e12824d627ba7b27f167862fc48eb40525d960f40f46df7af8f1602R57
is newly added for enterprise test.

Reply:
I dont know what happened in github, but here it is https://github.com/Kong/kubernetes-testing-framework/pull/104/files#diff-937a1ace7e12824d627ba7b27f167862fc48eb40525d960f40f46df7af8f1602